### ISSUE TITLE
Add arm64 images

### ISSUE THREAD

**Issue by mijdavis2** [hashicorp/docker-hub-images#102]
With the recent addition of arm64 cloud instances in providers like AWS, running images on arm architecture is becoming more and more attractive. Docker now also supports multi-arch image which make deploying anywhere much more convenient. It would be great to be able to run official hashicorp images on arm64 cloud instances.

**Comment by mogggggg** [comment]
Agreed, this would be a huge help - the binaries are already built!
